sdk/spring/CHANGELOG.md

```diff
@@ -1,49 +1,151 @@
 # Release History
 
 ## 4.0.0-beta.4 (Unreleased)
-- Support Spring Boot version: 2.5.5+ and 2.6.0 - 2.6.3.
-- Support Spring Cloud version: 2020.0.3+ and 2021.0.0.
+- Support Spring Boot version: 2.5.5+ and 2.6.0+.
+- Support Spring Cloud version: 2020.0.3+ and 2021.0.0+.
```

Comment:
How about write like this: 

> Supported Spring Boot version: `2.5.10`, `2.6.4`. **Note**: `2.5.x (x>10)` and `2.6.y (y>4)` should be supported, but it's not tested.
Supported Spring Cloud version: `2020.0.5`, `2021.0.1`. **Note**: `2020.0.x (x>5)` and `2021.0.y (y>1)` should be supported, but it's not tested.

Because:

1. `2021.0.0+` is confusing. Do `2021.0.99` and `2021.99.0` both belong to `2021.0.0+`?
2. `2021.0.99` is not released yet, we are not sure that our libraries support these versions before these versions release. For example: [azure-spring-boot:3.11.0 supports 2.5.4](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/azure-spring-boot/CHANGELOG.md#3110-2021-11-24), it does not support `spring-boot:2.5.5`. When we release `3.11.0`, it's possible that `spring-boot:2.5.5` is not released yet. We are not sure whether `azure-spring-boot:3.11.0` can support `spring-boot:2.5.5`.

`SpringCloudAzue` support `Spring Boot 2.5 and 2.6`, which does not mean `SpringCloudAzure:4.0.0` should support all patch versions of `SpringBoot 2.5 and 2.6`. We can release new `SpringCloudAzure` version to support latest SpringBoot patch version.